src/components/AccordionMenu/styles.module.scss

```diff
@@ -15,40 +15,6 @@
 }
 
 .itemButton {
-  background-color: $light-gray;
-  display: flex;
```

Comment:
There are a few properties that have not been brought into the styles for the new `Button` component. It looks okay to me, but we should be careful that we're not losing any styles that are important.